Emby.Naming/ExternalFiles/ExternalPathParserResult.cs

```diff
@@ -1,17 +1,17 @@
-namespace Emby.Naming.Subtitles
+namespace Emby.Naming.ExternalFiles
 {
     /// <summary>
     /// Class holding information about subtitle.
```

Comment:
Update subtitle references to be more general, perhaps "external track file" or something along those lines to cover audio and subtitles.